src/api/v1/community_businesses/register.ts

```diff
@@ -0,0 +1,97 @@
+import * as Hapi from 'hapi';
+import * as Boom from 'boom';
+import { omit } from 'ramda';
+
+import { CommunityBusinesses, Users, CbAdmins } from '../../../models';
+import { isChildOrganisation } from '../prerequisites';
+import { RegisterCommunityBusinessesRequest } from '../types';
+import { response, cbPayload } from './schema';
+import {
+  userName,
+  email,
+} from '../users/schema';
+import { EmailTemplate } from '../../../services/email/templates';
+
+export default [
+  {
+    method: 'POST',
+    path: '/community-businesses/register',
+    options: {
+      description: 'Register a new community businesses and invite admin user',
+      auth: {
+        strategy: 'standard',
+        access: {
+          scope: ['organisations_details-child:write'],
+        },
+      },
+      validate: {
+        payload: {
+          ...omit(['name'], cbPayload),
```

Comment:
You can just use `renameKeys` on this, no? But also, is this even necessary? There's no naming conflict, and (IMO) it's fairly clear that `name` would mean the name of the CB 🤷🏼‍♂️

Reply:
`cb/register` payload also take a value for adminName. The naming convention here is to explicitly state which "name" is being referenced.

Can make this rename keys, had issues with ramda in the past so by default try to stay away from it in ts files 🤷🏽‍♀️